idb/postgres/internal/encoding/types.go

```diff
@@ -111,26 +111,32 @@ type specialAddresses struct {
 type baseOnlineAccountData struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
-	VoteID          crypto.OneTimeSignatureVerifier `codec:"A"`
-	SelectionID     crypto.VRFVerifier              `codec:"B"`
-	StateProofID    merklesignature.Verifier        `codec:"C"`
-	VoteFirstValid  basics.Round                    `codec:"D"`
-	VoteLastValid   basics.Round                    `codec:"E"`
-	VoteKeyDilution uint64                          `codec:"F"`
+	VoteID          crypto.OneTimeSignatureVerifier `codec:"vote"`
+	SelectionID     crypto.VRFVerifier              `codec:"sel"`
+	StateProofID    merklesignature.Verifier        `codec:"stprf"`
+	VoteFirstValid  basics.Round                    `codec:"voteFst"`
+	VoteLastValid   basics.Round                    `codec:"voteLst"`
+	VoteKeyDilution uint64                          `codec:"voteKD"`
 }
 
 type baseAccountData struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
-	Status                     basics.Status `codec:"a"`
-	AuthAddr                   crypto.Digest `codec:"b"`
-	TotalAppSchemaNumUint      uint64        `codec:"c"`
-	TotalAppSchemaNumByteSlice uint64        `codec:"d"`
-	TotalExtraAppPages         uint32        `codec:"e"`
-	TotalAssetParams           uint64        `codec:"f"`
-	TotalAssets                uint64        `codec:"g"`
-	TotalAppParams             uint64        `codec:"h"`
-	TotalAppLocalStates        uint64        `codec:"i"`
+	Status              basics.Status `codec:"onl"`
+	AuthAddr            crypto.Digest `codec:"spend"`
```

Comment:
There is a query that needs to be change `b` to `spend`.

Reply:
terrifying, thanks

Reply:
This still isn't resolved, you need to revert this change in `postgres.go`:
```
  if len(opts.EqualToAuthAddr) > 0 {
    whereParts = append(whereParts, fmt.Sprintf("a.account_data ->> 'b' = $%d", partNumber))
    whereArgs = append(whereArgs, encoding.Base64(opts.EqualToAuthAddr))
    partNumber++
  }
```

Reply:
thanks! i left that in a stash, updated now